### PR TITLE
Added download script for example server

### PR DESCRIPTION
**Pull request by JonasHelming**
fixed #175

Signed-off-by: Jonas Helming <jhelming@eclipsesource.com>

### CODE REVIEW

**Comment by JonasHelming** [comment]
Does not work yet due to https://github.com/eclipse-glsp/glsp/issues/176

File: README.md [hunk 1]
```diff
@@ -12,16 +12,20 @@ See [our project website](https://www.eclipse.org/glsp/documentation/#workflowov
...
 ```
 yarn install
 ```
-
-Then, you need to build and run the Workflow Diagram Server example. See [`glsp-server`](https://github.com/eclipse-glsp/glsp-server#building-the-workflow-diagram-example-server) for instructions on how to build and run the server.
-
+Next, download a pre-built version of the Workflow Example Diagram Server and start it (replace X.X.X with the current version, the download script will print out the correct command on the console):
+```
+yarn download:exampleServer
+java -jar org.eclipse.glsp.example.workflow-X.X.X-SNAPSHOT-glsp.jar org.eclipse.glsp.example.workflow.launch.ExampleServerLauncher --port=8081 --websocket
```
Comment (planger) [inline README.md]:
```suggestion
java -jar org.eclipse.glsp.example.workflow-X.X.X-SNAPSHOT-glsp.jar org.eclipse.glsp.example.workflow.launch.ExampleServerLauncher
```

File: example/workflow/scripts/download.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+const mavenRepository = "https://oss.sonatype.org/content/repositories/snapshots/";
+const groupId = "org.eclipse.glsp.example";
+const artifactId = "org.eclipse.glsp.example.workflow";
+const version = "0.9.0";
+const classifier = "glsp";
+
+console.log("Downloading latest version of the Workflow Example Java Server from the maven repository...");
+download({ groupId, artifactId, version, classifier, isSnapShot: true }, downloadDir, mavenRepository)
+    .then(() => console.log("Download completed. Start the server using this command: \njava -jar org.eclipse.glsp.example.workflow-"
+        + version + "-SNAPSHOT-glsp.jar org.eclipse.glsp.example.workflow.launch.ExampleServerLauncher --port=8081 --websocket\n\n"
```
Comment (planger) [inline example/workflow/scripts/download.ts]:
```suggestion
        + version + "-SNAPSHOT-glsp.jar org.eclipse.glsp.example.workflow.launch.ExampleServerLauncher\n\n"
```

**Review by planger — CHANGES_REQUESTED** [review]
I wasn't able to retest yet, but from a first look the parameters that are passed to the server launch are wrong.
We only (have to) use websocket for the standalone mode as the browser connects directly to the GLSP server. For all other cases, we use plain sockets as a backend process connects to the GLSP server.
It could be that removing those arguments address https://github.com/eclipse-glsp/glsp/issues/176

**Comment by JonasHelming** [comment]
Thanks, that was the issue!

**Review by planger — APPROVED** [review]
Thanks, looks good to me! :+1:

**Pull request merged**